db/upgrade.php

```diff
@@ -325,17 +325,19 @@ function xmldb_tool_excimer_upgrade($oldversion) {
         upgrade_plugin_savepoint(true, 2022041301, 'tool', 'excimer');
     }
 
-    if ($oldversion < 2022042000) {
+    if ($oldversion < 2022042700) {
```

Comment:
The 2022042000 version is baked in and already out in the wild (even if only on a very small number of sites). Best practice is to never ever modify a previous versions upgrade step as you can't know which sites have or havent run that step. Instead you need to make future steps work with whatever state may be present.

So leave the existing one alone and make a new if ($oldversion < 2022042700) { section